src/libraries/Microsoft.Win32.Primitives/tests/Microsoft.Win32.Primitives.Tests.csproj

```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>$(NetCoreAppCurrent)</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent)-Windows_NT;$(NetCoreAppCurrent)-Unix;$(NetCoreAppCurrent)</TargetFrameworks>
```

Comment:
What is the RIDless configuration supposed to test? Shouldn't this be browser?

Reply:
The reasons behind choosing the ridless over browser 
1) There is nothing specific to browser in the default configuration
2) It can be used on other platforms which will fall into not unix not windows bucket.